k8s/namespaces/sscs/sscs-hearings-api/ithc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: sscs-hearings-api
+  namespace: sscs
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/sscs/hearings-api:prod-2d34469-20220325150900 #{"$imagepolicy": "flux-system:sscs-hearings-api"}
```

Comment:
Image should be in base Helm Release if not overriding prod image.  Make similar changes in applicable environments.